tests/components/homekit_controller/common.py

```diff
@@ -228,30 +227,40 @@ async def device_config_changed(hass, accessories):
     pairing._accessories_state = AccessoriesState(
         accessories_obj, pairing.config_num + 1
     )
-
-    discovery_info = zeroconf.ZeroconfServiceInfo(
-        host="127.0.0.1",
-        addresses=["127.0.0.1"],
-        hostname="mock_hostname",
-        name="TestDevice._hap._tcp.local.",
-        port=8080,
-        properties={
-            "md": "TestDevice",
-            "id": "00:00:00:00:00:00",
-            "c#": "2",
-            "sf": "0",
-        },
-        type="mock_type",
+    pairing._async_description_update(
```

Comment:
FIXME - the pairing doesn't have a description, so it needs to see 2 descriptions to detect a change. Need to decide what to do here.

Reply:
(See https://github.com/Jc2k/aiohomekit/pull/156)

Reply:
Fixed.